csm/core/services/users.py

```diff
@@ -254,15 +254,15 @@ async def _validation_for_update_by_normal_user(self, user_id: str, loggedin_use
         """
         current_password = new_values.get(const.CSM_USER_CURRENT_PASSWORD, None)
         if user_id.lower() != loggedin_user_id.lower():
-            raise CsmPermissionDenied("Non super user cannot change other user",
+            raise CsmPermissionDenied("Non admin user cannot change other user",
                                     USERS_MSG_PERMISSION_DENIED, user_id)
         
         if not current_password:
-            raise InvalidRequest("Current password is required",
+            raise InvalidRequest("value for current_password is required",
                                     USERS_MSG_UPDATE_NOT_ALLOWED, user_id)
```

Comment:
first letter of "value" should be capital

Reply:
Corrected